pkg/apis/acid.zalan.do/v1/operator_configuration_type.go

```diff
@@ -79,18 +88,22 @@ type LoadBalancerConfiguration struct {
 	ReplicaDNSNameFormat      config.StringTemplate `json:"replica_dns_name_format,omitempty"`
 }
 
+// AWSGCPConfiguration defines the configuration for AWS
+// TODO complete Google Cloud Platform (GCP) configuration
 type AWSGCPConfiguration struct {
 	WALES3Bucket string `json:"wal_s3_bucket,omitempty"`
 	AWSRegion    string `json:"aws_region,omitempty"`
 	LogS3Bucket  string `json:"log_s3_bucket,omitempty"`
 	KubeIAMRole  string `json:"kube_iam_role,omitempty"`
 }
 
+// OperatorDebugConfiguration defines the configration of debug
```

Comment:
`defines options for debug mode`?

Reply:
Do these types with useful names really need comments? :dancer:

Reply:
Yes, it's questionable. The original purpose was to silence the golint, since it complains about missing comments. I thought we could do this in a more meaningful way,